### PR TITLE
cluster-ui: clean up tooltip components

### PR DESCRIPTION
**Pull request by elkmaster**
before: in `cluster-ui` package we used legacy versions of
tootips based on antd and with a bunch of code duplicates and
hacks.

after: used `ui-components` tooltip component, removed unneeded
code and styles. also on process of work find out that barchats
legend tooltips missing styles - added them.

use tooltip component from `ui-components` library instead local
versions based on antd. to achieve this `ui-componets/Tooltip`
also needed some updates:
 - provide new style for table title usage (it have same color
   scheme as `default` but different typographic style)
 - add ability to hover on tooltip for clickable elements inside
 - other minor style updates and fixes

as side effect of this work there is a fix of dev process, updates of 
storybook and webpack config was needed to run build
of `cluster-ui` with linked local `ui-components`.

### CODE REVIEW

File: packages/storybook-ui-components/package.json [hunk 1]
```diff
@@ -24,12 +24,12 @@
...
-    "@storybook/addon-knobs": "^5.3.18",
-    "@storybook/addon-links": "^5.3.18",
-    "@storybook/addons": "^5.3.18",
-    "@storybook/react": "^5.3.18",
+    "@storybook/addon-actions": "^6.1.21",
+    "@storybook/addon-docs": "^6.1.21",
+    "@storybook/addon-knobs": "^6.1.21",
+    "@storybook/addon-links": "^6.1.21",
+    "@storybook/addons": "^6.1.21",
+    "@storybook/react": "^6.1.21",
```
Comment (nathanstilwell) [inline packages/storybook-ui-components/package.json:32]: 😄 The design team will be so happy we got a Storybook upgrade!

**Review by nathanstilwell — APPROVED** [review]
LGTM. This is exciting to see use reducing the number of Tooltips in the repo! I'm also pretty stoked about the Storybook upgrade.

**Comment by elkmaster** [comment]
blocked by https://github.com/cockroachdb/ui/pull/271

**Comment by thtruo** [comment]
I just noticed this was merged (woo!), but am still seeing unreadable tooltips in staging. Here's an example:

Screen Recording 2021-04-20 at 02 54 PM

Is this a known issue?

**Pull request merged**